### PR TITLE
Create Son of Sam

### PR DESCRIPTION
**Pull request by WndyCity**

### CODE REVIEW

File: Added Son of Sam [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "project": "Son of Sam",
+    "tags": [
+        "Son of Sam"
+    ],
+    "policies": [
+        "44d3143cf17a410b4875d55ff9b78e8acd3c430276945af0c8c3a091",
```
Comment (itsmestale) [inline Added Son of Sam:7]: remove comma

File: Rain Dance Eye [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "project": “Rain Dance Eye“,
+    "tags": [
+        “Rain Dance Eye“
+    ],
+    "policies": [
+        "ac2a609e41dcfc352244c67471f341a6e19661ce4b6e04509ee062cc",
```
Comment (itsmestale) [inline Rain Dance Eye:7]: remove comma

**Review by itsmestale — CHANGES_REQUESTED** [review]

File: Added Son of Sam (hunk 1 above)
Comment (WndyCity) [reply Added Son of Sam]: This is now done. Thank you.

File: Rain Dance Eye (hunk 2 above)
Comment (WndyCity) [reply Rain Dance Eye]: This is now done, thank you.

**Pull request merged**